ckanext/googleanalytics/controller.py

```diff
@@ -156,3 +159,53 @@ def _post_analytics(
     def dump(self, resource_id):
         self._post_analytics(c.user, "Resource", "Download", resource_id)
         return DatastoreController.dump(self, resource_id)
+class GAOrganizationController(OrganizationController):
+    def _post_analytics(self, user, request_obj_type, request_function, request_id):
+        if config.get('googleanalytics.id'):
+            data_dict = {
+                "v": 1,
+                "tid": config.get('googleanalytics.id'),
+                "cid": hashlib.md5(user).hexdigest(),
+                # customer id should be obfuscated
+                "t": "event",
+                "dh": c.environ['HTTP_HOST'],
+                "dp": c.environ['PATH_INFO'],
+                "dr": c.environ.get('HTTP_REFERER', ''),
+                "ec": "CKAN Organization Page View",
+                "ea": request_obj_type+request_function,
+                "el": request_id,
+            }
+            plugin.GoogleAnalyticsPlugin.analytics_queue.put(data_dict)
+    def read(self, id, limit=20):
+       self._post_analytics(c.user,"Organization","View",id)
```

Comment:
@psbothra Why are we using the organization id here instead of title? We're using title for packages and resources.